projects/assets/src/translations/en/common.ts

```diff
@@ -210,4 +210,14 @@ export const common = {
     showPassword: 'Show password',
     hidePassword: 'Hide password',
   },
+  chatMessaging: {
+    charactersLeft: 'characters left: {{count}}',
+    addNewMessage: 'Add New Message',
+    send: 'Send',
+    uploadFile: 'Upload File',
+    textLabel: '{{author}}. {{text}} at {{date}}',
```

Comment:
maybe rename this to title label or informationLabel, to specify its function rather than type

Reply:
sure.